app/scripts/services/import-service.js

```diff
@@ -334,7 +334,10 @@
           .filter(canImport)
           .map(function (name) {
             return function () {
-              return self.createFile(directory, name, files[name]);
+              return self.createFile(directory, name, files[name])
+                .then(function (file) {
+                  return ramlRepository.saveFile(file);
```

Comment:
If the file fails to save, what do we do?

I would make it so that if a file fails to save, the import continues, and maybe show an error that says: "Some of the imported files could not be saved, re-try saving all files.", or something like that